pkg/cloudprovider/aws/amifamily/al2.go

```diff
@@ -78,7 +79,15 @@ func (a AL2) containerRuntime(instanceTypes []cloudprovider.InstanceType) string
 // DefaultBlockDeviceMappings returns the default block device mappings for the AMI Family
 func (a AL2) DefaultBlockDeviceMappings() []*v1alpha1.BlockDeviceMapping {
 	return []*v1alpha1.BlockDeviceMapping{{
-		DeviceName: aws.String("/dev/xvda"),
-		EBS:        &defaultEBS,
+		DeviceName: a.EphemeralBlockDevice(),
+		EBS:        &DefaultEBS,
 	}}
 }
+
+func (a AL2) EphemeralBlockDevice() *string {
+	return aws.String("/dev/xvda")
+}
+
+func (a AL2) EphemeralBlockDeviceOverhead() resource.Quantity {
+	return resource.MustParse("2Gi")
```

Comment:
Probably worth some more discussion how we calculate this overhead.  I'm starting to think we should take a static minValue or a percentage of the size of the `EphemeralBlockDevice`.

Reply:
It will vary by the AMI, not sure of the best way to do it.  For AL2 it looks to be 2.8G.  Maybe just bump it to 5G for all for now? @bwagner5 may have an opinion.

```
/dev/nvme0n1p1   20G  2.8G   18G  14% /
```

Reply:
I bumped this to 5GB for now.